src/settings/MobileAppSettings.js

```diff
@@ -30,6 +30,7 @@ class MobileAppSettings extends Settings {
     const currentLanguage = this.get(SETTINGS_KEYS.CURRENT_LANGUAGE);
     setCurrentLanguage(currentLanguage);
```

Comment:
Might be missing some info, but I can't think of a situation where you would want to set these to different languages. If that's the case, I think could make this a bit tidier by calling them all in `setCurrentLanguage`?